odata-to-abstract-sql.ometajs

```diff
@@ -755,30 +791,23 @@ var generateShortAliases = function(clientModel) {
 	var traverseNodes = function(str, node) {
 		_.each(node, function(value, key) {
 			if (key === '$suffix') {
-				var lowerCaseResourceName = str + value;
-				var origResourceName = _.find(resourceNames, function(resourceName) {
-					return resourceName.toLowerCase() === lowerCaseResourceName;
+				var lowerCaseAliasPart = str + value;
+				var origAliasPart = _.find(aliasParts, function(aliasPart) {
+					return aliasPart.toLowerCase() === lowerCaseAliasPart;
 				});
-				shortAliases[origResourceName] = origResourceName.slice(0, str.length);
+				var length = str.length;
+				if (str[str.length - 1] === ' ') {
+					// Add an extra character so we don't end on a space
+					length += 1;
```

Comment:
I think you meant `length -= 1` here? Otherwise I'm confused as to what this bit does.